internal/providers/pluginfw/products/sharing/data_shares_acc_test.go

```diff
@@ -39,47 +39,47 @@ func TestUcAccDataSourceShares(t *testing.T) {
 			}
 		}
 
-		resource "databricks_table" "mytable" {
+		resource "databricks_sql_table" "mytable" {
 			catalog_name = databricks_catalog.sandbox.id
 			schema_name = databricks_schema.things.name
 			name = "bar"
 			table_type = "MANAGED"
-			data_source_format = "DELTA"
+			warehouse_id = "{env.TEST_DEFAULT_WAREHOUSE_ID}"
+			properties = {
+				"delta.enableDeletionVectors" = "false"
+			}
```

Comment:
is this because deletion vectors are being enabled automatically for workspaces? if so, would it be better to enable history sharing on delta sharing instead?